.ci/packaging.groovy

```diff
@@ -3,7 +3,7 @@
 @Library('apm@current') _
 
 pipeline {
-  agent { label 'ubuntu && immutable' }
+  agent none
```

Comment:
will it work the `gitCheckout` ?

Reply:
I have tested and it works

Reply:
However, I have launched it on the CI to test it also in the CI https://beats-ci.elastic.co/job/Beats/job/packaging/job/PR-19559/2/console, I'll merge it when it finished